src/Automation/Automation/help/Set-AzAutomationWebhook.md

```diff
@@ -35,7 +35,23 @@ This command disables a webhook named Webhook01 in the Automation account named
 
 Modifies a webhook for an Automation runbook. (autogenerated)
 
+### Example 3
+```powershell
+PS C:\>Set-AzAutomationWebhook -Name "Webhook01" -ResourceGroup "ResourceGroup01" -AutomationAccountName "AutomationAccount01" -RunOn 'Windows'
+```
+
+This command sets the run on value for the webhook and forces the runbook to be run on a Hybrid Worker group called Windows.
+
+### Example 4
+```powershell
```

Comment:
One example can only contain one powershell code block

Reply:
Resolved - also removed the example that had no code listed